saleor/order/utils.py

```diff
@@ -702,7 +697,22 @@ def get_valid_shipping_methods_for_order(
         listing = listing_map.get(method.id)
         shipping_method_data = convert_to_shipping_method_data(method, listing)
         if shipping_method_data:
-            valid_methods.append(shipping_method_data)
+            all_methods.append(shipping_method_data)
+    return all_methods
+
+
+def get_valid_shipping_methods_for_order(
+    order: Order,
+    shipping_channel_listings: Iterable["ShippingMethodChannelListing"],
+    manager: "PluginsManager",
+) -> List[ShippingMethodData]:
+    """Return a list of shipping methods according to Saleor's own business logic.
+
+    The resulting methods are not yet filtered by plugins.
```

Comment:
Is it correct? See lines 717 and 718. Look like we already filter out them

Reply:
I haven't noticed since I didn't change the logic and left the docstring as it was. I agree, the description is confusing since it filters excluded methods.